### DISCUSSION TITLE
1.0.9.3

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#53] [Обновления]
Решил не тянуть с выходом обновления до последнего и выпустить, хоть и не доделанное до конца, но, обновление. Поскольку, список изменений и так получился довольно большой, да и, сейчас, опять не сильно много времени, чтобы уделять его обновлению скрипта. В 1.0.9.4 постараюсь добавить остальные сайты (vimeo, gag, возможно тикток. С меньшей вероятностью coursera, facebook, twitter, udemy)

Изменения:
- Добавлена поддержка смены видео на m.twitch.tv (Рекомендую использовать только совместно с автопереводом, ибо без него кнопка перевода требует 2-ого нажатия. Не знаю как это исправить :с)
- Добавлена кнопка загрузки файла перевода
- Исправлена ошибка из-за который при нажатие на элементы меню расширения воспроизводилось видео (Если вам не нравится это изменение - отпишите в отдельной issue, если многие будут против - верну)
- Исправлена ошибка из-за который при нажатие на слайдер внутри меню расширения воспроизводилось видео
- Уменьшено значение "анимации" первого открытия окна настроек расширения
- Добавлена частичная поддержка **[VK](https://vk.com/video)** (Рекомендую использовать только совместно с автопереводом, ибо без него кнопка перевода требует 2-ого нажатия. Не знаю как это исправить :с)
- Добавлена поддержка **[Invidious](https://github.com/iv-org/invidious)** (Добавлено для доменов yewtu.be, inv.vern.cc и других)
- Добавлена поддержка Twitch Player (player.twitch.tv)
- Добавлено округление числа в кастомном слайдере громкости (для чтобы не было чисел с множеством знаков после запятой)
- Добавлена поддержка Pornhub & Pornhub Embed
- Добавлена поддержка YouTube NoCookie (youtube-nocookie.com)
- Добавлена поддержка XVideos (Перевод получается супер странный, но ладно... + в хромоподобных браузерах не отображается картинка Алисы из-за ограничений ссылок со стороны сайта)
- Стили слайдера теперь применяются принудительно, независимо от стилей самого сайта

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.0.9.2...1.0.9.3

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.0.9.3'>1.0.9.3</a>.</em>

**Comment by FiLinX** [comment]
"Добавлена поддержка Pornhub & Pornhub Embed"     - да ладно? серьёзно чтоль??  а где там перевод то нужон?  ))
ну яндых и удивляет ))

**Reply by ilyhalight** [reply]
> "Добавлена поддержка Pornhub & Pornhub Embed" - да ладно? серьёзно чтоль?? а где там перевод то нужон? )) ну яндых и удивляет ))

Ахах. Там ещё вот это есть

>     * Добавлена поддержка XVideos
